src/components/Settings/SettingsProfile.js

```diff
@@ -0,0 +1,68 @@
+import {Button, Image, Text, TextInput, View} from "react-native";
+import {viewStyles} from "../../styles/settings/settings";
+import {launchImageLibrary} from "react-native-image-picker";
+import React from "react";
+
+const SettingsProfile = ( { settings, onSettingsModified } ): React.Node => {
+  const profileSource =
+    settings.removeProfilePhoto || !settings.icon ?
+      require( "./../../images/profile.png" ) :
+      { uri: settings.newProfilePhoto ? settings.newProfilePhoto.uri : settings.icon};
+
+  const onImageSelected = ( response ) => {
+    if ( response.didCancel ) {return;}
+    onSettingsModified( { ...settings, newProfilePhoto: response.assets[0], removeProfilePhoto: false } );
+  };
+
+  return (
+    <>
+      <Text>Profile Picture</Text>
+      <View style={viewStyles.row}>
+        <Image
+          style={viewStyles.profileImage}
+          source={profileSource}
+        />
+        <View style={viewStyles.column}>
+          <Button title="Upload New Photo" onPress={() => launchImageLibrary( {}, onImageSelected )} />
+          <Button title="Remove Photo" onPress={() => onSettingsModified( { ...settings, removeProfilePhoto: true } )} />
+        </View>
+      </View>
+      <View style={viewStyles.column}>
+        <Text>Username</Text>
```

Comment:
One thing I'm noticing throughout Settings is that we're not pulling in the user's existing information. We should display the user's current username, email, bio, etc. so they can see their user profile before making changes.

Reply:
Was related to v2 vs v1 API usage - now working

Reply:
Definitely loading everything that's available right now, with the caveat that there's an [API problem](https://github.com/inaturalist/iNaturalistAPI/issues/324) with returning email.